app/main.py

```diff
@@ -1,2 +1,144 @@
+import math
+from collections.abc import Hashable
+from typing import Any, Dict, Generator, Iterable, Tuple
```

Comment:
If `Hashable` is used only for annotation, the import should be done from `from typing import Hashable `.

Reply:
Ok